apiserver/facades/client/client/status.go

```diff
@@ -374,11 +389,19 @@ type statusContext struct {
 	applications map[string]*state.Application
 
 	// remote applications: application name -> application
-	remoteApplications map[string]*state.RemoteApplication
-	relations          map[string][]*state.Relation
-	units              map[string]map[string]*state.Unit
-	latestCharms       map[charm.URL]*state.Charm
-	leaders            map[string]string
+	consumerRemoteApplications map[string]*state.RemoteApplication
```

Comment:
consumerRemoteApplications are the RemoteApplication entities in the consuming model. These correspond to the act of consuming an offer.
isConsumerProxy is true in the RemoteApplication entity in the offering model, ie a proxy for a consuming app in the consumer's model.